source/agora/node/Ledger.d

```diff
@@ -99,3 +129,71 @@ unittest
     assert(ledger.getLastBlock().tx == tx);
     assert(ledger.getLastBlock().header.tx_hash == tx.hashFull());
 }
+
+/// getBlocksFrom tests
+unittest
+{
+    import agora.common.crypto.Key;
+
+    scope ledger = new Ledger;
+    assert(ledger.getLastBlock() == getGenesisBlock());
+    assert(ledger.ledger.length == 1);
+
+    void generateBlocks (size_t count)
+    {
+        foreach (_; 0 .. count)
+        {
+            auto rand_pair = KeyPair.random();
+            auto input = Input(hashFull(42));
```

Comment:
Use the predicted height here ?

Reply:
> Use the predicted height

Sorry what?

Reply:
Well it's obv... Nvm
I originally wrote "use the index" to have a bit more randomness.
Then realized that if you call this function twice, you'll have twice the same set of txs.
Hence, use the height at which this tx is supposed to be included.

Reply:
> I originally wrote "use the index" to have a bit more randomness.

"Every time I hear about randomness in blockchain.."